cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -542,25 +578,19 @@ func (m *stateMachine) execute() (finished bool, err error) {
 func (m *stateMachine) mountBoot() (stateFunc, error) {
 	// use the disk we mounted ubuntu-seed from as a reference to find
 	// ubuntu-seed and mount it
-	partUUID, err := m.disk.FindMatchingPartitionUUID("ubuntu-boot")
-	if err != nil {
+	partUUID, findErr := m.disk.FindMatchingPartitionUUID("ubuntu-boot")
```

Comment:
side-note, but when we refactor FindMatchingPartitionUUID to return a full device path that can just be mounted/unlocked directly, then the `setFindState` can just set Device directly too

not for this PR, more for me to remember as I refactor the disks package soon